gpt_engineer/main.py

```diff
@@ -97,11 +112,13 @@ def main(
         azure_endpoint=azure_endpoint,
     )
 
-    input_path = Path(project_path).absolute()
-    print("Running gpt-engineer in", input_path, "\n")
+    path = Path(project_path).absolute()
+    print("Running gpt-engineer in", path, "\n")
+
+    workspace_path = path
 
-    workspace_path = input_path / "workspace"
-    project_metadata_path = input_path / ".gpteng"
+    project_metadata_path = path / ".gpteng"
+    input_path = project_metadata_path
```

Comment:
I'm not sure I agree with this.

Reply:
Unless I'm missing something, let's:
```suggestion
    input_path = path
```